content/javascript/concepts/arrays/terms/filter/filter.md

```diff
@@ -0,0 +1,119 @@
+---
+Title: ".filter()"
+Description: "Creates a new array with all elements that pass the test implemented by the provided function."
```

Comment:
Great metadata. The only thing I would change is: "implemented by" to "from" to keep it more simple.